hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/map/DefaultMapQueryMessageTask.java

```diff
@@ -48,4 +52,14 @@ protected void extractAndAppendResult(Collection<QueryResultRow> results, QueryR
         results.addAll(queryResult.getRows());
     }
 
+    protected void incrementOperationStat() {
```

Comment:
to obey DRY principle, maybe we can use this method for all message tasks by extracting it to a utility class.